pyproject.toml

```
@@ -22,6 +22,7 @@ dependencies = [
     "djangorestframework>=3.15.2",
     "django-storages[s3]>=1.14.4",
     "drf-spectacular>=0.27.2",
+    "bittensor>=7.3.1",
```

```suggestion
    "bittensor>=7.3.1;<7.4.0",
```
I wish they used semver properly, but they don't.